lib/twemoji.rb

```diff
@@ -249,7 +218,7 @@ def self.emoji_url(name)
       code = find_by_text(name)
 
       if options[:file_ext] == ".png"
-        File.join(options[:asset_root], options[:image_size], "#{code}.png")
+        File.join(options[:asset_root], "72x72", "#{code}.png")
```

Comment:
I don't think "72x72" is used elsewhere, but still it looks like it deserves to be a constant?